deep_speech_2/scorer.py

```diff
@@ -0,0 +1,62 @@
+"""External Scorer for Beam Search Decoder."""
```

Comment:
It would be better to have a new directory ./lm, and put data into ./lm/data,  and scorer.py into ./lm/lm_scorer.py.